lib/cli/options.js

```diff
@@ -210,7 +210,8 @@ module.exports.loadPkgRc = loadPkgRc;
  * 1. Command-line args
  * 2. RC file (`.mocharc.c?js`, `.mocharc.ya?ml`, `mocharc.json`)
  * 3. `mocha` prop of `package.json`
- * 4. default configuration (`lib/mocharc.json`)
+ * 4. `MOCHA_OPTIONS` environment variable.
+ * 5. default configuration (`lib/mocharc.json`)
```

Comment:
@JoshuaKGoldberg I'm not sure about the priority ordering here. I can't decide if env vars should be higher up or not.

Reply:
(Drive-by comment from a Yargs and Commander contributor, but not a Mocha user so do not have domain insights.)

Environment variables can be specified on the command-line, and I think are often treated as second in priority behind the command-line args.

Yargs specifies:
> Program arguments are defined in this order of precedence:
> 1. Command line args
> 2. Env vars
> 3. Config file/objects
> 4. Configured defaults

https://yargs.js.org/docs/#api-reference-envprefix

Reply:
@shadowspawn that's my gut feeling too after playing with it for a while. @JoshuaKGoldberg cool if I update the PR?

Reply:
Sounds good to me 🙂 thanks for the insights @shadowspawn!